snap/snapcraft.yaml

```diff
@@ -59,7 +59,10 @@ parts:
     - git+https://opendev.org/openstack/neutron-tempest-plugin.git@2.8.0
     - git+https://opendev.org/openstack/octavia-tempest-plugin.git@2.7.0
     - git+https://opendev.org/openstack/telemetry-tempest-plugin.git@2.2.0
-    - git+https://opendev.org/openinfra/python-tempestconf.git@3.5.0
+    - git+https://opendev.org/openstack/watcher-tempest-plugin.git@3.1.0
+    # Point to specific commit to include watcher-tempest-plugin discoverability feature
+    # until the commit is released.
+    - git+https://opendev.org/openinfra/python-tempestconf.git@550e43bb45855359264c9e3dd9130ad6219c6f2d
```

Comment:
I'm not sure how this will interact with the automation, but fine to include it now.  We can update automation later. :)

Note also that the automation will clobber the comment here.

Reply:
I see your point. What are your thoughts on automation updates required for these kind of changes?

Reply:
I'm not sure - I think this is fine for now - automation appears to pull the latest release of python-tempestconf.  So we can just hold off automated updates until this is released?

Reply:
@samuelallan72 do you have more any concern?

Reply:
@chanchiwai-ray Can you hold merging the PR as I am discussing with @aieri on better approaches to handle these kind of scenarios. I will bring the discussion in this PR once we have some direction

Reply:
@chanchiwai-ray no concerns from me, but agreed that we should hold off until we get the go-ahead from Andrea and Hemanth. :)

Reply:
I discussed with Andrea and we will merge this PR for now but in future will try to update automation to have manual-requirements have precedence over plugin requirements or tempest conf requirements.

@samuelallan72 Can you resolve the conversation and merge the PR, thanks